tasktiger/constants.py

```diff
@@ -0,0 +1,11 @@
+# constants pertaining to Redis keys
+REDIS_PREFIX = "t"
```

Comment:
This isn't used, nor is it needed (it's configurable, and the default value is defined in `tasktiger.tasktiger`.